mParticle-Radar/MPKitRadar.m

```diff
@@ -103,28 +102,77 @@ - (MPKitExecStatus *)didBecomeActive {
     return [[MPKitExecStatus alloc] initWithSDKCode:[MPKitRadar kitCode] returnCode:MPKitReturnCodeSuccess];
 }
 
-#pragma mark Events
+#pragma mark User attributes and identities
+
+- (NSString*) getMpId:(FilteredMParticleUser *)user {
+    if (user != nil && user.userId != nil && user.userId.integerValue != 0) {
+        return [user.userId stringValue];
+    } else {
+        return nil;
+    }
+}
+
+- (void)setMparticleMetadata:(NSString *)mpId {
+    if ([Radar getMetadata] != NULL) {
+        NSMutableDictionary *userMetadata = [[Radar getMetadata] mutableCopy];
+        [userMetadata setObject:mpId forKey:@"mParticleId"];
+        NSDictionary *userMetadataDict = [userMetadata copy];
```

Comment:
You may not need to do this because `NSMutableDictionary` is a subclass of `NSDictionary`. Can you just do
```
NSMutableDictionary *metadata = [[Radar getMetadata] mutableCopy];
[metadata setObject:mpId forKey:@"mParticleId"];
[Radar setMetadata:metadata];
```
?